docs/source/tutorial_metrics.md

```diff
@@ -421,11 +421,11 @@ please [file an issue on GitHub](https://github.com/facebookresearch/ParlAI/issu
 | ----------------------- | ------------ |
 | `accuracy`              | Exact match text accuracy |
 | `bleu-4`                | BLEU-4 of the generation, under a standardized (model-independent) tokenizer |
+| `clen`                  | Average length of context in number of tokens |
 | `clip`                  | Fraction of batches with clipped gradients |
-|`context_truncate`       | Ratio of samples' contexts being truncated per batch |
-|`context_length`         | Average length of context tokens per batch |
 | `ctpb`                  | Context tokens per batch |
 | `ctps`                  | Context tokens per second |
+| `ctrun`                 | Fraction of samples with some context truncation |
```

Comment:
nit: can we have it be `ctrunc`; this reads to me like `ct run` which is a bit confusing